includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -1,10 +1,18 @@
 <?php
 
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+use Wikia\Logger\WikiaLogger;
+use function GuzzleHttp\Psr7\build_query;
+
 class DesignSystemApiController extends WikiaApiController {
 	const PARAM_PRODUCT = 'product';
 	const PARAM_ID = 'id';
 	const PARAM_LANG = 'lang';
 	const PRODUCT_WIKIS = 'wikis';
+	const MEMC_PREFIX_FANDOM_STORE = 'DesignSystemCommunityHeaderModelClass::getFandomShopDataFromIntentX';
```

Comment:
Is this used elsewhere as a pattern? Why not keep this short and sweet? Also what if the function name changes.

Reply:
I saw this pattern being used with the class name and function.